test/helper.lua

```diff
@@ -509,4 +509,17 @@ function helpers.assert_timeout_error(value, message)
     error(err, 2)
 end
 
+function helpers.fflush_cluster_stdout(cluster, capture)
```

Comment:
Thank you for noticing.

I've renamed it:
fflush_cluster_stdout -> fflush_main_server_stdout
and now it calls eval from `main_server` not from `router` because in integration tests we run queries on the `main_server`.